boms/cloud-oss-bom/RELEASING.md

```diff
@@ -15,3 +15,29 @@ Create a PR for the release, and get it approved.
 
 Rapid project is [cloud-java-tools-cloud-opensource-java-bom-kokoro-release](
 http://rapid/cloud-java-tools-cloud-opensource-java-bom-kokoro-release).
+
+## Deleting a release
+
+Occasionally you may need to clean up after an aborted release, typically becaus the release script had
+problems. If so:
+
+1. Delete the release branch on Github.
+
+2. Fetch the tags in your local client:
+
+   `$ git fetch --tags --force`
```

Comment:
Let's use a fenced block here too for consistency.

Reply:
done